docs/templates/ADOC.tmpl

```diff
@@ -4,7 +4,7 @@
 
 ifeval::[{show-deprecation} == true]
 
-[[deprecation-note]]
+[[#deprecation-note-{{ .Timestamp }}]]
```

Comment:
```suggestion
[#deprecation-note-{{ .Timestamp }}]
```
`[[xx]]` is the legacy form for creating custom anchors. The new form is `[#xx]`